### PR TITLE
change code-server title

### PR DESCRIPTION
**Pull request by bpmct**
I noticed the description had it as "code-server," but I made a couple of changes to explain what code-server is and fix the heading. Totally understand if you want to keep the current description, this is just a suggestion :)

### CODE REVIEW

**Comment by FarazPatankar** [comment]
Thanks for the PR, @bpmct. Cool if we only update the description? Would be nice to keep the title consistent across all starters.

starters@2x

**Comment by bpmct** [comment]
that works!

File: examples/code-server/README.md [hunk 1]
```diff
@@ -8,6 +8,6 @@ tags:
   - ide
 ---
 
-# Code Server Example
+# code-server Example
 
 This example is maintained by the [Coder](https://coder.com/) team and the source code can be found [here](https://github.com/cdr/deploy-code-server).
```
Comment (1chiSensei) [inline examples/code-server/README.md:13]:
```suggestion
This example is maintained by the [Coder](https://coder.com/) team and the source code can be found [here](https://github.com/cdr/code-server).
```

Isn't that repo used to list on how to deploy `code-server`?

**Review by 1chiSensei — CHANGES_REQUESTED** [review]

File: examples/code-server/README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Code Server
+title: code-server
```
Comment (coffee-cup) [inline examples/code-server/README.md:2]: As per the discussion, going to keep the title Code Server :)
```suggestion
title: Code Server
```

**Pull request merged**